README.md

```diff
@@ -76,28 +76,28 @@ Cache update strategy. [More details about `updateStrategy`](docs/update-strateg
 > Default: `'changed'`.
 
 #### `externals: Array<string>`
-Allows to specify _external_ assets (assets which aren't generated by webpack) to be cached. If you don't change `caches` option then this should be enough to add _external_ assets to the cache. For other details and more advanced use cases see [`caches` docs](docs/caches.md).
+Allows you to specify _external_ assets (assets which aren't generated by webpack) that should be included in the cache. If you don't change the `caches` configuration option then it should be enough to simply add _external_ assets to the cache. For other details and more advanced use cases see [`caches` docs](docs/caches.md).
 
 > Default: `null`  
 > **Example value:** `['fonts/roboto.woff']`
 
 #### `excludes: Array<string | globs_pattern>`
-Excludes matched assets from being added to the caches. Exclusion is performed before _rewrite_ happens.  
+Excludes matched assets from being added to the cache. Exclusion is performed before _rewrite_ happens.  
```

Comment:
No problem, I will make it "caches" again, but link it to the respective option.